functional_tests/scylla_operator/test_functional.py

```diff
@@ -121,3 +121,21 @@ def test_mgmt_backup(tester, db_cluster):
     mgr_task = mgr_cluster.create_backup_task(location_list=[bucket_name, ])
     status = mgr_task.wait_and_get_final_status(timeout=54000, step=5, only_final=True)
     assert TaskStatus.DONE == status
+
+
+def test_listen_address(tester, db_cluster):
+    """
+    Issues: https://github.com/scylladb/scylla-operator/issues/484
+            https://github.com/scylladb/scylla/issues/8381
+    Fix commit: https://github.com/scylladb/scylla-operator/pull/529
+    """
+    all_errors = []
+    for node in db_cluster.nodes:
+        with node.remote_scylla_yaml() as scylla_yaml:
```

Comment:
remote_scylla_yaml is overriden in ScyllaPodContainer to address configmap, not a file on the node.

to fix that you need to find:
```
class BaseScyllaPodContainer(BasePodContainer):  # pylint: disable=abstract-method
    def restart(self):
        pass

    parent_cluster: ScyllaPodCluster

    @contextlib.contextmanager
    def remote_scylla_yaml(self) -> ContextManager:
```

Add there 

```
class BaseScyllaPodContainer(BasePodContainer):  # pylint: disable=abstract-method
    def restart(self):
        pass

    parent_cluster: ScyllaPodCluster

    def actual_scylla_yaml(self):
        return super().remote_scylla_yaml()

    @contextlib.contextmanager
    def remote_scylla_yaml(self) -> ContextManager:

```

And reuse actual_scylla_yaml in here, instead of remote_scylla_yaml

Reply:
done